server/channels/app/migrations.go

```
@@ -105,7 +105,9 @@ func (a *App) SetPhase2PermissionsMigrationStatus(isComplete bool) error {
 }
 
 func (a *App) DoEmojisPermissionsMigration() {
```

Let's return an error and check that. That ensures the error gets surfaced.